src/Sfa.Tl.Matching.Functions/MatchingServiceReport.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Sfa.Tl.Matching.Data.Interfaces;
+using Sfa.Tl.Matching.Domain.Models;
+using Sfa.Tl.Matching.Functions.Extensions;
+
+namespace Sfa.Tl.Matching.Functions
+{
+    public static class MatchingServiceReport
+    {
+        [FunctionName("GetMatchingServiceOpportunityReport")]
```

Comment:
Do Not Rename Azure Function Names as this will break Reports on PowerBI Side, this needs to be coordinated